Questionnaire/Questionnaire-RoD-YoungPersonAssentForm-Example.json

```diff
@@ -0,0 +1,301 @@
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "radio-button",
+                    "display": "Radio Button"
+                  }
+                ]
+              }
+            }
+          ],
+          "linkId": "consentQuestion4",
+          "text": "4. Do you understand it’s OK to say you don’t want to take part – but that your parent(s), or guardian who look after you, will make the final choice?",
+          "required": true,
+          "answerOption": [
+            {
+              "valueCoding": {
+                "display": "Yes"
+              }
+            },
+            {
+              "valueCoding": {
+                "display": "No"
+              }
+            }
+          ]
+        },
+        {
+          "type": "choice",
+          "extension": [
+            {
+              "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+              "valueCodeableConcept": {
+                "coding": [
+                  {
+                    "system": "http://hl7.org/fhir/questionnaire-item-control",
+                    "code": "radio-button",
+                    "display": "Radio Button"
+                  }
+                ]
+              }
+            }
+          ],
+          "linkId": "consentQuestion5",
+          "text": "5. Are you happy to take part?",
+          "required": true,
+          "answerOption": [
+            {
+              "valueCoding": {
+                "display": "Yes"
+              }
+            },
+            {
+              "valueCoding": {
+                "display": "No"
+              }
+            }
+          ]
+        }
+      ],
+      "type": "group",
+      "linkId": "declarationResponse",
+      "text": "Please indicate your choices below by ticking the appropriate box:",
+      "readOnly": true
+    },
+    {
+      "item": [
+        {
+          "type": "display",
+          "linkId": "NonWillingToConsent1",
+          "text": "• Don’t sign your name on this form"
+        },
+        {
+          "type": "display",
+          "linkId": "NonWillingToConsent2",
+          "text": "• Tell your parents and healthcare team how you feel, so they know"
+        }
+      ],
+      "type": "group",
+      "linkId": "guidanceNonWillingToConsent",
+      "text": "If ANY of your answers are ‘NO’, or you don’t want to take part:",
+      "required": true
+    },
+    {
+      "item": [
+        {
+          "type": "display",
+          "linkId": "WillingToConsent",
+          "text": "• Please write your name, signature, and today’s date here:"
+        }
+      ],
+      "type": "group",
+      "linkId": "guidanceWillingToConsent",
+      "text": "If ALL of your answers are ‘YES’:"
+    },
+    {
+      "item": [
```

Comment:
Could you add an item for the "Assent obtained remotely" box and make these items only mandatory/shown if this is unchecked?